Corrfunc/mocks/DDtheta_mocks.py

```diff
@@ -351,6 +353,278 @@ def DDtheta_mocks(autocorr, nthreads, binfile,
+        where the RA's are in [-180, 180.0]. For peace of mind, always supply
+        RA's in [0.0, 360.0].
+
+        Calculations are done in the precision of the supplied arrays.
+
+    DEC : array-like, real (float/double)
+        Array of Declinations for the first set of points. DEC's are expected
+        to be in the [-90.0, 90.0], but the code will try to fix cases where
+        the DEC's are in [0.0, 180.0]. Again, for peace of mind, always supply
+        DEC's in [-90.0, 90.0].
+        Must be of same precision type as RA.
+
+    verbose : boolean (default false)
+        Boolean flag to control output of informational messages
+
+    output_thetaavg : boolean (default false)
+        Boolean flag to output the average ``\theta`` for each bin. Code will
+        run slower if you set this flag.
+
+        If you are calculating in single-precision, ``thetaavg`` will
+        suffer from numerical loss of precision and can not be trusted. If you
+        need accurate ``thetaavg`` values, then pass in double precision arrays
+        for ``RA/DEC``.
+
+    isa: string, case-insensitive (default ``fastest``)
+        Controls the runtime dispatch for the instruction set to use. Options
+        are: [``fastest``, ``avx512f``, ``avx``, ``sse42``, ``fallback``]
+
+        Setting isa to ``fastest`` will pick the fastest available instruction
+        set on the current computer. However, if you set ``isa`` to, say,
+        ``avx`` and ``avx`` is not available on the computer, then the code
+        will revert to using ``fallback`` (even though ``sse42`` might be
+        available).
+
+        Unless you are benchmarking the different instruction sets, you should
+        always leave ``isa`` to the default value. And if you *are*
+        benchmarking, then the string supplied here gets translated into an
+        ``enum`` for the instruction set defined in ``utils/defs.h``.
+
+    max_cells_per_dim: integer, default is 100, typical values in [50-300]
+        Controls the maximum number of cells per dimension. Total number of
+        cells can be up to (max_cells_per_dim)^3. Only increase if ``rpmax`` is
+        too small relative to the boxsize (and increasing helps the runtime).
+
+    maxbinref: integer (default 3)
+        The maximum ``bin refine factor`` to use along each dimension. From
+        experience, values larger than 3 do not improve ``wp`` runtime.
+
+        Runtime of module scales as ``maxbinref^3``, so change the value of
+        ``maxbinref`` with caution.
+
+    nrepeats: integer (default 3)
+        Number of times to repeat the timing for an individual run. Accounts
+        for the dispersion in runtimes on computers with multiple user
+        processes.
+
+    return_runtimes: boolean (default ``false``)
+        If set, also returns the array of runtimes.
+
+    Returns
+    -------
+    (RA1, DEC1, RA2, DEC2) : tuple of integers
```

Comment:
This is probably a typo -- a tuple of two values `(RA_bin_ref, DEC_bin_ref)` corresponding to the fastest average runtime (and lowest scatter in case of ties) should be returned

Reply:
Yes, this has indeed been a typo. I have changed it to something like 
```
(nRA, nDEC): tuple of integers
```
instead corresponding to the fastest average runtime (plus lowest scatter in the case of a tie) to be returned.